src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = isMarried ? (
+    `${partnerName} is my ${sex === 'f' ? ('husband') : ('wife')}`
+  ) : (
+    `I am not married`
+  );
```

Comment:
It's bad practice to use nested ternary operators.
Could you try to separate this logic? For example:
```
const partnerText = `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'`;
const statusText = isMarried ? partnerText : 'I am not married';
```